src/components/dialogs/network-modifications/shunt-compensator/characteristics-pane/characteristics-form.js

```diff
@@ -66,41 +118,102 @@ export const CharacteristicsForm = ({
         />
     );
 
-    const susceptancePerSectionField = (
+    const maxSusceptanceField = (
         <FloatInput
-            name={SUSCEPTANCE_PER_SECTION}
-            label={'ShuntSusceptancePerSection'}
+            name={MAX_SUSCEPTANCE}
+            label={'MaxShuntSusceptance'}
             adornment={SusceptanceAdornment}
             previousValue={previousValues?.bperSection}
             clearable={isModification}
         />
     );
 
+    const switchedOnSusceptanceField = (
+        <FloatInput
+            name={SWITCHED_ON_SUSCEPTANCE}
+            label={'SwitchedOnMaxSusceptance'}
+            adornment={SusceptanceAdornment}
+            clearable={isModification}
+            formProps={{
+                disabled: true,
+            }}
+        />
+    );
+
     const characteristicsChoiceField = (
         <RadioInput
             name={CHARACTERISTICS_CHOICE}
             options={Object.values(CHARACTERISTICS_CHOICES)}
         />
     );
 
-    const characteristicsChoice = useWatch({
-        name: CHARACTERISTICS_CHOICE,
-    });
+    const handleSwitchedOnValue = useCallback(
+        (linkedSwitchedOnValue, SWITCHED_ON_FIELD) => {
+            if (
+                ![
+                    sectionCount,
+                    maximumSectionCount,
+                    linkedSwitchedOnValue,
+                ].includes(null)
+            ) {
+                trigger(SECTION_COUNT).then((isValid) => {
+                    if (isValid) {
+                        setValue(
+                            SWITCHED_ON_FIELD,
+                            (linkedSwitchedOnValue / maximumSectionCount) *
+                                sectionCount
+                        );
+                    } else {
+                        setValue(SWITCHED_ON_FIELD, null);
+                    }
+                });
+            } else {
+                setValue(SWITCHED_ON_FIELD, null);
+                clearErrors(SECTION_COUNT);
+            }
+        },
+        [maximumSectionCount, sectionCount, clearErrors, setValue, trigger]
+    );
+
+    useEffect(() => {
+        if (
+            characteristicsChoice === CHARACTERISTICS_CHOICES.Q_AT_NOMINAL_V.id
+        ) {
+            handleSwitchedOnValue(maxQAtNominalV, SWITCHED_ON_Q_AT_NOMINAL_V);
+        } else if (
+            characteristicsChoice === CHARACTERISTICS_CHOICES.SUSCEPTANCE.id
+        ) {
+            handleSwitchedOnValue(maxSusceptance, SWITCHED_ON_SUSCEPTANCE);
+        }
+    }, [
+        maxQAtNominalV,
+        maxSusceptance,
+        characteristicsChoice,
+        handleSwitchedOnValue,
+    ]);
 
     return (
         <>
             <Grid container spacing={2}>
+                {gridItem(maximumSectionCountField, 4)}
+                {gridItem(sectionCountField, 4)}
                 {gridItem(characteristicsChoiceField, 12)}
                 {characteristicsChoice ===
```

Comment:
```suggestion
                {characteristicsChoice ===
                    CHARACTERISTICS_CHOICES.SUSCEPTANCE.id && (
                    <Grid item container spacing={2}>
                        {gridItem(maxSusceptanceField, 4)}
                        {gridItem(switchedOnSusceptanceField, 4)}
                    </Grid>
                )}
```
you can maybe regroup the fields to avoid having the check on every field. 
and you can do the same for "CHARACTERISTICS_CHOICES.Q_AT_NOMINAL_V.id"

Reply:
Agreed, it's way cleaner